app/models/category.rb

```diff
@@ -0,0 +1,4 @@
+class Category < ActiveRecord::Base
+  validates_presence_of :name, uniqueness: true
+  # validates :name, uniqueness: true
```

Comment:
Can we take the comment out? Also, validates_presence_of already validates uniqueness.